e2e/less/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "e2e-less",
+    "private": true,
+    "devDependencies": {
+        "@bazel/less": "file:../../dist/bin/packages/less/npm_package"
```

Comment:
this can be "latest" - after we push the first time, the e2e directory will be self-contained

Reply:
Gotcha. So is it expected for the e2e builds to fail until this package is deployed?